.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][<jira-issue-id>]: <short-description-of-the-pr> --->
+
+Version:
+<!--- What version or versions of OpenShift does this apply to? --->
+
+Issue:
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+Link to docs preview:
+<!--- Add direct links to the Netlify preview build. --->
+
+Description:
+
+<!--- For more information about contributing to OpenShift documentation, see:
+https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/contributing.adoc --->
+
+<!--- Squash to one commit before submitting your PR for peer review -->
+
+<!--- IMPORTANT: All documentation changes must be verified by a QE team associate before merging.
+For more information about verifying your content, see:
+https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#verification-of-your-content --->
+
+<!--- To ask for peer review:
+- Go to the #forum-docs-review channel in the CoreOS Slack workspace https://coreos.slack.com/archives/C025XCRRH28
+- Ping the peer review squad (@peer-review-squad) with a link to your PR
+- Include the size of your PR by using the :size-<size-of-pr> emoji --->
```

Comment:
I honestly don't think that this is critical enough to ask people outside our team to do